chain/chain/src/runtime/mod.rs

```diff
@@ -842,8 +842,8 @@ impl RuntimeAdapter for NightshadeRuntime {
 
             // Take a single transaction from this transaction group
             while let Some(tx_peek) = transaction_group_iter.peek_next() {
-                // Stop adding transactions if the size limit would be exceeded
-                if checked_feature!("stable", WitnessTransactionLimits, protocol_version)
+                // WitnessTransactionLimits: Stop adding transactions if the size limit would be exceeded
```

Comment:
is there a specific reason that you want to keep feature names in comments? eg. for still being able to search through the code after this PR?

Reply:
Yeah, that was the idea, since stateless validation is such a large feature, it makes sense to know what exactly are we checking when we are doing a protocol version check.